test/sys-info.ts

```diff
@@ -287,6 +296,7 @@ describe("SysInfo unit tests", () => {
 			});
 
 			it("is null when OS is not Mac", async () => {
+				process.env[PROGRAM_FILES] = PROGRAM_FILES_ENV_PATH;
```

Comment:
it is bad practice to modify `process.env` in a test, as the change will be kept for all other tests.
You can use lodash's cloneDeep:
```TypeScript
const originalProcessEnv = process.env;
process.env = _.cloneDeep(process.env);
process.env[PROGRAM_FILES] = PROGRAM_FILES_ENV_PATH;
// tests code is here
process.env = originalProcessEnv;
// assert here - it is important to have the asserts AFTER resetting the `process.env`,
// so even if the assert fails, you'll be the other tests will never know about your newly introduced environment variable.
```